src/routes/transactions/mappers/common/twap-order.mapper.spec.ts

```diff
@@ -427,7 +430,14 @@ describe('TwapOrderMapper', () => {
       .build();
     const fullAppData = JSON.parse(fakeJson());
 
-    mockSwapsRepository.getOrder.mockResolvedValueOnce(part2);
+    mockSwapsRepository.getOrder.mockImplementation(
+      async (_chainId: string, orderUid: string) => {
+        if (orderUid === part2.uid) {
+          return Promise.resolve(part2);
+        }
+        return Promise.reject(new NotFoundException());
+      },
+    );
```

Comment:
I encountered issues in the new test cases that were due to incorrectly calculated order UIDs. As such, I adjusted these mocks to be more specific.